test/astar.cc

```diff
@@ -1679,6 +1679,81 @@ TEST(BiDiAstar, test_recost_path) {
   }
 }
 
+// TODO(nils): this test fails currently, because bidir A* has a problem with 2 shortcuts between the
+// same nodes: https://github.com/valhalla/valhalla/issues/4609
```

Comment:
this disabled test is the reason why the reported coverage is so low BTW.

also, good to mention: if I run this on master (with the appropriate edits to make sure it's 2 shortcuts with different superseded edges between A and E), it fails too.